pkg/receiver/mysqlreceiver/receiver.go

```diff
@@ -0,0 +1,70 @@
+package mysqlreceiver 
+ 
+import (
+	"context"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/component"
+	"go.uber.org/zap"
+	"go.opentelemetry.io/collector/model/pdata"
+)
+
+//var records map[int]string
+
+type mySQLReceiver struct {
+	sqlclient client
+	logger    *zap.Logger
+	config    *Config
+	//data	  map[int]string
+	consumer consumer.Logs
+}
+
+func newMySQLReceiver (logger *zap.Logger, conf *Config, next consumer.Logs) (component.LogsReceiver, error) {
+
+	return &mySQLReceiver{
+		consumer: next,
+		logger: logger,
+		config: conf,
+		//data: records,
+	},nil	
+}
+
+func (m *mySQLReceiver) Start(ctx context.Context, host component.Host) error {
+	sqlclient := newMySQLClient(m.config)
+
+	err := sqlclient.Connect()
+	if err != nil {
+		return err
+	}
+	m.sqlclient = sqlclient
+
+	data,err := m.sqlclient.getRecords()
```

Comment:
missing space, between `data` and `err`, please get familiar with linters for go